test/test_s3_utils.py

```diff
@@ -220,3 +220,23 @@ def test_unzip_s3_to_s3_2(integrated_s3_info):
 
     objs = integrated_s3_info['s3Obj'].s3_read_dir(prefix)
     assert objs.get('Contents', None)
+
+
+def test_unzip_s3_to_s3(integrated_s3_info):
```

Comment:
This test name duplicates an earlier name. I probably `test_unzip_s3_to_s3_do_not_store` or something like that would help?